Description: Define an application-specific module path
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 online-python-tutor (3~git59a556a-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: Olivier Berger <obergix@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- online-python-tutor-3~git59a556a.orig/v3/web_exec.py
+++ online-python-tutor-3~git59a556a/v3/web_exec.py
@@ -14,9 +14,11 @@
 
 import cgi
 import json
-import pg_logger
 import sys
 
+# Make sure that the app's modules are available, as where they got installed by the Debian package
+sys.path.insert(0, '/usr/share/online-python-tutor-backend/')
+import pg_logger
 
 # set to true if you want to log queries in DB_FILE 
 LOG_QUERIES = False
